src/main/java/uk/gov/companieshouse/confirmationstatementapi/eligibility/impl/CompanyOfficerValidation.java

```diff
@@ -19,29 +20,42 @@
 
     private final OfficerService officerService;
 
+    private Boolean officerValidationFlag;
+
     @Autowired
-    public CompanyOfficerValidation(OfficerService officerService){
+    public CompanyOfficerValidation(OfficerService officerService, Boolean officerValidationFlag){
         this.officerService = officerService;
+        this.officerValidationFlag = officerValidationFlag;
     }
 
     @Override
     public void validate(CompanyProfileApi companyProfileApi) throws EligibilityException, ServiceException {
         LOGGER.info("Validating Company Officers for: {}", companyProfileApi.getCompanyNumber());
+        if (!Boolean.TRUE.equals(officerValidationFlag)) {
+            LOGGER.debug("OFFICER VALIDATION FEATURE FLAG off skipping validation");
+            return;
+        }
         var officers = officerService.getOfficers(companyProfileApi.getCompanyNumber());
-        var officerCount = getOfficerCount(officers.getItems(), officers.getActiveCount());
+        var officerCount = getOfficerCount(officers.getItems());
         if (officerCount != null && officerCount > 1) {
             LOGGER.info("Company Officers validation failed for: {}", companyProfileApi.getCompanyNumber());
             throw new EligibilityException(EligibilityStatusCode.INVALID_COMPANY_APPOINTMENTS_MORE_THAN_ONE_OFFICER);
         }
         LOGGER.info("Company Officers validation passed for: {}", companyProfileApi.getCompanyNumber());
     }
 
-    public Long getOfficerCount(List<CompanyOfficerApi> officers, Long activeCount) {
-        for(CompanyOfficerApi i: officers) {
-            if (i.getOfficerRole().getOfficerRole().contains("secretary")) {
-                activeCount--;
+    public Long getOfficerCount(List<CompanyOfficerApi> officers) {
+        Long officerCount = 0L;
+        for(CompanyOfficerApi officer: officers) {
+            var role = officer.getOfficerRole();
+            if (role == OfficerRoleApi.DIRECTOR || role == OfficerRoleApi.NOMINEE_DIRECTOR || role == OfficerRoleApi.CORPORATE_DIRECTOR) {
+                officerCount++;
             }
         }
-        return activeCount;
+        return officerCount;
+    }
+
+    public void setOfficerValidationFlag(Boolean officerValidationFlag) {
```

Comment:
as the flag is now set via a constructor arg. This setter is uneeded

Reply:
Im still using this in the test class as an easy way to change the flag for test coverage

Reply:
Can you not just create a new instamce for the test with the required flag value passed into the constructor?